action.yml

```diff
@@ -0,0 +1,55 @@
+name: 'Build Plugin'
+description: 'Build GOG Galaxy integration on current OS'
+
+inputs:
+  src:
+    description: 'Location of directory containing manifest.json and essential plugin files'
+    required: true
+    default: 'src'
+  requirements:
+    description: 'Location of Python requirements file'
+    required: true
+    default: 'requirements/app.txt'
```

Comment:
Instead of having these be different from repo to repo, would it be better to standardise these locations across the repos?

Reply:
In most plugins it is exactly this, so it is a standardization, but not requirement. I don't want to force any convention  -- especially if it requires change in the plugin code -- it is beyond the org scope.

Reply:
Oh yeah that makes alot of sense.